app/main/views/search.py

```diff
@@ -228,8 +229,26 @@ def supplier_search():
         seller_type = details.get('seller_type', {})
 
         is_recruiter = details.get('is_recruiter', False)
+        # consultant only badge to show in seller catalogue
+        if is_recruiter == 'false' and seller_type and 'recruiter' not in seller_type.keys():
+            seller_type['recruitment'] = False
+            seller_type['consultant_only'] = True
+            seller_type['recruiter_both'] = False
+            seller_type['recruiter_only'] = False
+
+        # recruiter only badge to show in seller catalogue
         if is_recruiter == 'true' and seller_type and 'recruiter' not in seller_type.keys():
-            seller_type['recruitment'] = True
+            seller_type['recruitment'] = False
```

Comment:
Not sure why we have two for recruiter (legacy data?) but should this be `True`?

Reply:
It is legacy data that we have and seem to use previously for the badges. You can find instance in the frontend. It seems that for the badges we should use the new format `seller_type['recruiter_only']` .

Reply:
I have removed it from the code.